.gitignore

```diff
@@ -10,6 +10,7 @@
 /bin/tool/
 /include/
 /lib*/
+/build/
```

Comment:
Ah, should I remove that?  I still have a lot of old scripts around that still uses that old convention from older Urho3D days.  ^.^